### ISSUE TITLE
[Tutorial] Virtualbox

### ISSUE THREAD

**Issue by pulkitkrishna00** [InstallerLegacy/setup.virtualhub.eu.org#12]
- Homepage: <https://www.virtualbox.org/>

Information:
- Package available (From Developer and Distribution both)

Status:

- [x] Tutorial
- [ ] Video

Depends on: #4

**Comment by pulkitkrishna00** [comment]
Added tutorial in f7cf0f4.

**Comment by pulkitkrishna00** [comment]
Closing this, as it has been decided that a separate issue will be created for video.
